linters/.markdownlint.json

```diff
@@ -0,0 +1,33 @@
+{
+  "comment": "Enable all rules, and then disable selective ones https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md",
```

Comment:
i would prefer (just like our eslint config) to not "enable all", but to always explicitly define every single possible rule in our config.